src/renderer/components/table-submenu.jsx

```diff
@@ -33,15 +33,20 @@ export default class TableSubmenu extends Component {
   }
 
   renderSubMenuHeader() {
+    const { itemsByTable, table } = this.props;
+
     const title = this.state.collapsed ? 'Expand' : 'Collapse';
     const cssClass = this.state.collapsed ? 'right' : 'down';
-
+    const cssStyle = { ...STYLE.header };
+    if (! itemsByTable || !itemsByTable[table] || itemsByTable[table].length === 0) {
```

Comment:
Since you gonna fix that. Could you also replace `itemsByTable[table].length === 0` with `!itemsByTable[table].length`? Is the way you usually use it in the project.